src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const NUMBER_OF_DAYS = days;
```

Comment:
`days` already comes as a variable from the function arguments, so there is no need to create a new variable for it.